apps/src/templates/courseOverview/AssignCourse.js

```diff
@@ -0,0 +1,189 @@
+import React, { Component, PropTypes } from 'react';
+import ReactDOM from 'react-dom';
+import Radium from 'radium';
+import $ from 'jquery';
+import color from "@cdo/apps/util/color";
+import i18n from '@cdo/locale';
+import ProgressButton from '@cdo/apps/templates/progress/ProgressButton';
+import BaseDialog from '@cdo/apps/templates/BaseDialog';
+import AssignCourseConfirm from './AssignCourseConfirm';
+
+const styles = {
+  button: {
+    marginBottom: 10
+  },
+  icon: {
+    fontSize: 24,
+    // we want our icon text to be a different size than our button text, which
+    // requires we manually offset to get it centered properly
+    position: 'relative',
+    top: 3
+  },
+  dropdown: {
+    border: `1px solid ${color.charcoal}`,
+    position: 'absolute',
+    marginTop: -10
```

Comment:
These are good questions. My instinct is that this isn't something we need to think about immediately, but it might be good to have a plan for the future. @poorvasingal might have thoughts.